### ISSUE TITLE
Incorrect PDB parsing of static variables

### ISSUE THREAD

**Issue by 0x1F9F1** [Vector35/binaryninja-api#1660]
Binary Ninja Version: 2.0.2144-dev, f704503e
Platform: Windows 10 Version 1903

The PDB parser loads static variables as functions, and at the wrong address

Given a static variable like
```
static int Foo;
```
If it is actually located at 0x10065df8, the PDB parser loads it at 0x20065df8 (image base seems to be added twice), and as a function.
```
Attempting to add function not backed by file: 0x20065df8
```
```
>>> bv.symbols['Foo']
<SymbolType.FunctionSymbol: "Foo" @ 0x20065df8>
```

**Comment by CouleeApps** [comment]
A quick and slightly messy workaround for this is rebasing the binary to 0x0, as when it adds the image base twice, it will not change.

**Comment by plafosse** [comment]
Fixed in 2.2.2517
